packages/injected-wallets/src/injectedWallets.ts

```diff
@@ -1,8 +1,7 @@
 import type { Injected, InjectedWindow, InjectedAccount } from '@polkadot/extension-inject/types';
-import type { Account, BaseWallet, BaseWalletProvider, WalletMetadata } from '@polkadot-onboard/core';
 import type { Signer } from '@polkadot/types/types';
+import { Account, BaseWallet, BaseWalletProvider, WalletMetadata, WalletType } from '@polkadot-onboard/core';
```

Comment:
import as type. We would want the lib to have zero dependencies on polkadot.js for tree shaking. These are not needed at runtime so should be only imported as types.